docs/sphinx/source/whatsnew/v0.8.1.rst

```diff
@@ -13,6 +13,8 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
+* Added :py:func:`pvlib.temperature.ross` for cell temperature modeling using
+  only NOCT. (:pull:`1045`)
```

Comment:
and add yourself to contributors below. I also forgot to add myself for the expanded asv tests.

Reply:
I'll add